spring-boot-project/spring-boot-dependencies/pom.xml

```diff
@@ -140,6 +140,8 @@
 		<nio-multipart-parser.version>1.1.0</nio-multipart-parser.version>
 		<pooled-jms-version>1.0.3</pooled-jms-version>
 		<postgresql.version>42.2.5</postgresql.version>
+		<!-- need to take care that this version harmonizes with micrometer ones -->
```

Comment:
I think we have a maven plugin enabled that takes care of that. We should drop the comment in the POM regardless because we generally avoid them in the POM files.